lib/container.ts

```diff
@@ -19,27 +19,27 @@ import { streamToBuffer } from './util';
  * should be provided as:
  * 	src/main.ts
  */
-interface ChangedFiles {
-	updated: string[];
-	added: string[];
-	deleted: string[];
+export interface ChangedFiles {
+	updated?: string[];
+	added?: string[];
+	deleted?: string[];
```

Comment:
It seems weird to allow null or an empty list.

Reply:
That's a fair point, I'll just make them required again then.